change note: Disallow certain application options to be provided via query params (#4187)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-23-4-disallow-certain-application-options-to-be-provided-via-query-params.md

---
date: ""

> title: Disallow certain application options to be provided via query params

Reviewer: ```suggestion
title: Disallowed certain application options to be provided via query parameters
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65991
version: 1023.23.4
---

> Application options can be provided via query parameters to Web SDK based UI applications for testing purposes.

Reviewer: ```suggestion
For testing purposes, application options can be provided via query parameters to Web SDK-based UI applications.
```

> We've now excluded some these applicaion options to be providable via query parameters.

Reviewer 2: Maybe?
```suggestion
We've now excluded some these application options to be provideable via query parameters to avoid CSS-injection via this parameters.
```

> We've now excluded some these applicaion options to be providable via query parameters.

Reviewer: ```suggestion
Some of these application options have now been excluded from being provided via query parameters.
```

> Some of these application options have now been excluded from being provided via query parameters.

Reviewer: ```suggestion
Some of these application options have now been excluded from being made available via query parameters.
```

> Some of these application options have now been excluded from being made available via query parameters.

Reviewer: Should we add a reason? Security?

Reviewer 2: yes, let's simply say for security reasons but not clearly about, which stuff we fix.

Author: ```suggestion
Some of these application options have now been excluded from being made available via query parameters for security reasons.
```
